apps/worker/src/transaction_publisher/publisher.service.ts

```diff
@@ -102,7 +108,8 @@ export class TransactionPublisherService extends BaseConsumer implements OnAppli
         delay: blockDelay,
       });
     } catch (error) {
-      this.logger.error(error);
+      // @ts-ignore
+      this.logger.error('Hello world....', error, error?.stack);
```

Comment:
This looks like test code?
```suggestion
      this.logger.error('Unknown error encountered', error, error?.stack);
```